LiskHQ/lisk-mobile
# Update npm dependencies - Closes #1094

### What was the problem?
This PR resolves #1094

### How was it solved?
Updated dependencies with high vulnerabilities

### How was it tested?
- Build the app on simulator and real device on iOS and android
- Run tests to make sure everything passes too
